src/extension/templates/templates.ts

```diff
@@ -166,6 +168,20 @@ export default class Templates {
         vscode.tasks.executeTask(buildTask);
       }
     }
+
+    function populateLanguageArgs() {
+      switch (languageCode) {
+        case "python":
+          if (language.tasks) {
+            for (const task of language.tasks) {
+              if (task.command == "neo3-boa") {
+                task.args.push(mainFile);
+              }
+            }
+          }
+          break;
+      }
+    }
```

Comment:
After talking with another programmer, the code for putting the contract name in args ended up going inside `templates.ts` instead of `languages.ts`. If, by some reason, another language also needs to put additional dynamic information this method could be used too.